fil-proofs-tooling/src/bin/benchy/stacked.rs

```diff
@@ -145,8 +156,13 @@ where
                 wall_time: replication_wall_time,
                 return_value: (pub_inputs, priv_inputs),
             } = measure(|| {
-                let (tau, (p_aux, t_aux)) =
-                    StackedDrg::<H, Sha256Hasher>::replicate(&pp, &replica_id, &mut data, None)?;
+                let (tau, (p_aux, t_aux)) = StackedDrg::<H, Sha256Hasher>::replicate(
+                    &pp,
+                    &replica_id,
+                    &mut data,
+                    None,
+                    Some(config.clone()),
```

Comment:
given that the `StoreConfig` is always the same that gets passed in, how about we just pass the `cache_path` and construct the `StoreConfig` in the methods?

Reply:
@cryptonemo did you forget about this one, or just not yet done?

Reply:
Note: I think this applies to all methods, `replicate`, `extract_all`, etc. Can just pass the cache_path through and construct the configs in the places needed.

Reply:
We need the size from the config in some places, so unless we can always get that back, I'm not sure.  Otherwise, sounds fine.

Reply:
fair enough, lets just leave it for now